js/capture.js

```diff
@@ -3,8 +3,16 @@
 * third-party requests to storage.
 */
 const capture = {
-  init() {
+  async init() {
     this.addListeners();
+    // get Disconnect Entity List from shavar-prod-lists submodule
+    let whiteList
+      = await fetch('/shavar-prod-lists/disconnect-entitylist.json');
```

Comment:
Ok; using `.then` chaining currently; what options do I have to do this? You mentioned using an absolute URL as a web extension resource -- how would I determine that? Submitting current form which uses `.then`.